models/eprop_readout.cpp

```diff
@@ -313,23 +308,22 @@ nest::eprop_readout::compute_error_signal_cross_entropy_loss( const long& lag )
 void
 nest::eprop_readout::handle( DelayedRateConnectionEvent& e )
 {
-  size_t rport = e.get_rport();
-  const long delay = e.get_delay_steps();
-  long i = 0;
-
-  assert( rport < 3 );
-
-  std::vector< unsigned int >::iterator it = e.begin();
+  const size_t rport = e.get_rport();
+  assert( rport < SUP_RATE_RECEPTOR );
 
+  long i = 0;
```

Comment:
```suggestion
  long i = e.get_rel_delivery_steps( kernel().simulation_manager.get_slice_origin() );
```
With the changes in this PR running the regression task fails (while the classification task works). By changing the above line as suggested both work. Would that be maybe a valid solution?